src/gleam/erlang/os.gleam

```diff
@@ -63,3 +63,48 @@ pub external fn set_env(name: String, value: String) -> Nil =
 ///
 pub external fn unset_env(name: String) -> Nil =
   "gleam_erlang_ffi" "unset_env"
+
+/// Run a command in a command shell.
+///
+/// This function returns the output of the command as a String.
+///
+/// ## Examples
+///
+///    > command("ls")
+///    "Code\nDesktop\nDocuments\nMusic\nPictures\nVideos"
+///
+///    > command("ls -a")
+///    ".\n..\n.asdf\n.config\nCode\nDesktop\nDocuments\nMusic\nPictures\nVideos"
+///
+pub external fn command(String) -> String =
```

Comment:
I've been playing around with `cmd` and I'm not happy with the API it provides. There's no way to determine if a command failed or not short of parsing that string, which I think makes it largely useless for anything other than hacky scripts that can fail silently.


```erl
1> os:cmd("wefkjhwef").
"/bin/sh: line 1: wefkjhwef: command not found\n"
```

Reply:
Taking a step back, so we think that `os` _should_ have a `command` function to run arbitrary commands? The alternative would be to prefer single-purpose functions for what folks may intuitively reach for the shell to do (e,g. `file.list_directory` instead of `ls`, `file.make_directory` instead of `mkdir`), etc, and have the developer use Ports directly if they want to run a tool (e.g. `rustc` or `npm` or `gleam`).

Reply:
I think it'd be nice to have, but if we can't offer a good API it'd be better to defer until the time comes when we can. It's going to be hard to change anything we add to this library in future so best be conservative with what we add.

Reply:
I'm going to punt on the `command` function for now, will consider cycling back to it later.